File: Rust/src/ABC086/a_product.rs
pub fn run(a: i32, b: i32) -> &'static str {
    if (a * b) % 2 == 0 {
        "Even"
    } else {
        "Odd"
    }
}

#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn test() {
        assert_eq!("Even", run(3, 4));
        assert_eq!("Odd", run(3, 3));
    }
}

File: Erlang/AtCoder/abc086/a_product.erl
-module(a_product).
-export([main/0]).

% AとBの商を2で割り、商が0かどうかを返す
a_product(A, B) ->
  case ((A * B) rem 2) of
    0 -> "Even";
    _ -> "Odd"
  end.

main() ->
  io:format("~p ~n", [a_product(3, 4) =:= "Even"]),
  io:format("~p ~n", [a_product(1, 21) =:= "Odd"]).
